bedrock/mozorg/models.py

```diff
@@ -129,3 +131,42 @@ def blog_title(self):
     @property
     def blog_link(self):
         return settings.BLOG_FEEDS[self.blog_slug]['url']
+
+    @classmethod
+    def parse_feed(cls, feed_id, feed_options):
```

Comment:
They all could. I just couldn't find a better place for them, and adding them to the model class made imports cleaner. I could move them to a utils.py, but I think that the models file imports from utils, and so I can't import the model there w/o some inline thing... I'm conflicted on the best location, but this seems okay. I can move it again tho.